web/src/utils/unique-array.ts

```diff
@@ -1 +1,2 @@
-export const uniqueArray = <T>(arr: Array<T>): Array<T> => [...new Set(arr)];
+export const uniqueArray = <T>(arr: Array<T>): Array<T> =>
+  [...new Set(arr)].sort();
```

Comment:
It seems that you changed at the shared place.
Will this affect all places where this function is being used? Is that ok?

Reply:
yes, I confirmed that this one only is used in related places we want. And besides, having a sorted set is a good point as well, so I think it's okay to make change like this.

Reply:
sure, I rename it as sortedSet 🙆‍♀️